lib/mapping/MapReaderH3M.h

```diff
@@ -48,7 +48,7 @@ class MapReaderH3M
 	PlayerColor readPlayer();
 	PlayerColor readPlayer32();
 
-	void readBitmaskBuildings(std::set<BuildingID> & dest, std::optional<FactionID> faction);
+	std::set<BuildingID> readBitmaskBuildings(std::optional<FactionID> faction);
```

Comment:
What's the reason behind this change?

Reply:
These are not mandatory changes. Rolled back.